template2.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="template2.css" />
+    <script
+      src="https://kit.fontawesome.com/c66455b3cd.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Template 2</title>
+  </head>
+
+  <body>
+    <!-- container id added -->
+    <div class="container" id="container">
+      <div class="head">
+        <div class="left">
+          <h1 id="FName" style="font-weight: bold">Korina</h1>
+          <h1 id="LName">Villanueva</h1>
+          <p id="address">123 anywhere st. any city</p>
+        </div>
+        <div class="right">
+          <p id="phone"><i class="fa-solid fa-phone-volume"></i></p>
+          <p id="email"><i class="fa-solid fa-envelope"></i></p>
+          <p id="github"><i class="fa-solid fa-location-dot"></i>github.com</p>
+          <p id="linkedIn"><i class="fa-brands fa-linkedin"></i>linkedin id</p>
+          <p id="website"><i class="fa-solid fa-globe"></i>website.com</p>
+        </div>
+      </div>
+      <div class="main">
+        <div class="half-1">
+          <div class="about margin">
+            <h2>About Me</h2>
+            <p id="aboutme"></p>
+          </div>
+          <hr />
+          <div class="skills margin" id="skill-data">
```

Comment:
skills margin is not valid name pls use valid naming property

Reply:
multiple classes added here